navigator_node:
  ros__parameters:
    robot_radius: 0.1 # robotâ€™s radius for collision checking with the environment
    goal_tolerance: 0.1 # error tolerated by the end point of the path to the goal
    use_path_smoother: True
    max_planning_bounds: [20, 20]

    map_topic: /projected_map
